src/Domain.LinnApps/PurchaseOrders/PurchaseOrder.cs

```diff
@@ -10,7 +10,7 @@ public class PurchaseOrder
     {
         public string Cancelled { get; set; }
 
-        public IEnumerable<PurchaseOrderDetail> Details { get; set; }
+        public IList<PurchaseOrderDetail> Details { get; set; }
```

Comment:
Using IList for these so that I can use the .Add method it gives over IEnumerable in the domain service for when a new detail/delivery is added on update, like here https://github.com/linn/purchasing/blob/c37a38e7dabff4017103c4ac9e4344c40515f3e6/src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs#L124